src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -22,10 +20,14 @@
 import org.broadinstitute.hellbender.utils.read.AlignmentUtils;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 import org.broadinstitute.hellbender.utils.read.ReadUtils;
+import org.broadinstitute.hellbender.utils.recalibration.covariates.Covariate;
+import org.broadinstitute.hellbender.utils.recalibration.covariates.StandardCovariateList;
 
 import java.io.Serializable;
 import java.util.Arrays;
 
+import static org.broadinstitute.hellbender.utils.MathUtils.roundToNDecimalPlaces;
```

Comment:
i like them because they remove clutter and makes code easier to read,  but i dont care that much